.data/vSMR_WIP_areas.txt

```diff
@@ -64,24 +70,8 @@ COORD:N052.39.03.987:E000.33.29.030
 COORD:N052.38.20.422:E000.33.18.178
 COORD:N052.38.20.120:E000.33.22.443
 
-EGOV RWY (01/19 North) - MIL AIP SUP 014/2024 UFN
-COORD:N053.15.00.955:W004.32.19.548
-COORD:N053.15.27.404:W004.32.14.767
-COORD:N053.15.27.489:W004.32.17.933
-COORD:N053.15.02.345:W004.32.22.443
-
-EGOV RWY (01/19 South) - MIL AIP SUP 014/2024 UFN
-COORD:N053.14.58.627:W004.32.19.962
-COORD:N053.14.35.667:W004.32.24.200
-COORD:N053.14.35.891:W004.32.27.378
-COORD:N053.15.00.156:W004.32.22.868
-
-EGOV ASP1 - MIL AIP SUP 015/2024 UFN
-COORD:N053.15.21.847:W004.32.10.829
-COORD:N053.15.18.345:W004.32.11.249
-COORD:N053.15.15.153:W004.32.12.044
-COORD:N053.15.11.135:W004.32.12.546
-COORD:N053.15.10.830:W004.32.12.014
-COORD:N053.15.10.500:W004.32.08.640
-COORD:N053.15.10.944:W004.32.08.995
-COORD:N053.15.20.607:W004.32.07.471
+EGHI Stand 12 - AIP SUP 046/2025 UFN
```

Comment:
https://nats-uk.ead-it.com/cms-nats/opencms/en/Publications/aip-supplements/EG_Sup_2025_046_en.pdf refers to EGHH TWY A not EGHI Stand 12, which was https://www.aurora.nats.co.uk/htmlAIP/Publications/2024-05-02/html/eSUP/EG-eSUP-2024-046-en-GB.html